src/functions-and-arrays.js

```diff
@@ -1,19 +1,65 @@
 // Iteration #1: Find the maximum
-
+const maxOfTwoNumbers = (number1, number2) => {
+  if (number1 > number2) {
+    return number1
+  } else {
+    return number2;
+  }
+}
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+function findLongestWord(words) {
+  let longestWord = '';
+  if (words.length == '') {
```

Comment:
Your condition on line 13 is incorrect because the length of an array is always a number. What you have in your compare right now is an empty string, it should be

`if(words.length === 0) {...}`